.gitignore

```diff
@@ -1,2 +1,4 @@
 build/
 node_modules/
+
+h/static/scripts/coverage
```

Comment:
Can we move this to a top level `coverage/` directory? I strongly prefer to keep the source tree and build/test artifacts separated.